microservice/src/main/java/nearsoft/skytouch/microservice/repository/impl/ProductRepositoryImpl.java

```diff
@@ -0,0 +1,70 @@
+package nearsoft.skytouch.microservice.repository.impl;
+
+
+import nearsoft.skytouch.common.model.Product;
+import nearsoft.skytouch.microservice.repository.ProductRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.ParameterMode;
+import javax.persistence.PersistenceContext;
+import javax.persistence.StoredProcedureQuery;
+import javax.transaction.Transactional;
+import java.util.LinkedList;
+import java.util.List;
+
+@Repository
+public class ProductRepositoryImpl implements ProductRepository {
```

Comment:
I'd like you to compare this approach with the followed by your team mates (Dilan and Sergio)

Reply:
Ok i will ckeck their repos thanks